controller/controller.go

```diff
@@ -548,20 +589,27 @@ func NewProvisionController(
 	controller.claimQueue = workqueue.NewNamedRateLimitingQueue(rateLimiter, "claims")
 	controller.volumeQueue = workqueue.NewNamedRateLimitingQueue(rateLimiter, "volumes")
 
-	if controller.createProvisionedPVBackoff == nil {
-		// Use linear backoff with createProvisionedPVInterval and createProvisionedPVRetryCount by default.
-		if controller.createProvisionedPVInterval == 0 {
-			controller.createProvisionedPVInterval = DefaultCreateProvisionedPVInterval
-		}
-		if controller.createProvisionedPVRetryCount == 0 {
-			controller.createProvisionedPVRetryCount = DefaultCreateProvisionedPVRetryCount
-		}
-		controller.createProvisionedPVBackoff = &wait.Backoff{
-			Duration: controller.createProvisionedPVInterval,
-			Factor:   1, // linear backoff
-			Steps:    controller.createProvisionedPVRetryCount,
-			Cap:      controller.createProvisionedPVInterval,
+	if controller.createProvisionerPVLimiter != nil {
```

Comment:
should it be default someday? is there any performance hit? I guess it does not matter as long as csi-provisioner uses it.

Reply:
I'd say eventually it should be default, but after a long deprecation period. As you wrote, csi-provisioner is going to use it and that's what matters.